### DISCUSSION TITLE
Windows Store vs Installer?

### DISCUSSION THREAD

**Discussion by iSupremacy** [rocksdanister/lively#2192] [Q&A]
Just found the app today amazing amazing amazing.
Any difference between the Microsoft Store and the system installer versions thank you?
I have the Store version right now before i knew.
cheers

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions

In terms of functions mostly the same, the store version gets updated in the background via Microsoft Store and does not require admin rights to install.
